src/Container.tsx

```diff
@@ -242,7 +236,7 @@ export const Container = React.memo(
       )
 
       useAnimatedReaction(
-        () => headerHeight.value,
+        () => headerHeight,
```

Comment:
Does this actually work right? Can reanimated trigger a reaction based on a react state change?

Reply:
Yea, it does work, I added logs to validate.